app/controllers/users/registrations_controller.rb

```diff
@@ -31,47 +31,11 @@ def create
       end
     end
 
-    def edit
-      analytics.track_pageview
-      @update_user_profile_form = UpdateUserProfileForm.new(resource)
-    end
-
-    def update
-      @update_user_profile_form = UpdateUserProfileForm.new(resource)
-
-      if @update_user_profile_form.submit(user_params)
-        process_successful_update(resource)
-      else
-        clean_up_passwords resource
-        render :edit
-      end
-    end
-
     def destroy_confirm
     end
 
     protected
 
-    def process_successful_update(resource)
-      process_updates(resource)
-      bypass_sign_in resource
-    end
-
-    def process_updates(resource)
-      updater = UserFlashUpdater.new(@update_user_profile_form, flash)
-
-      updater.set_flash_message
-
-      if @update_user_profile_form.mobile_changed?
-        analytics.track_event('User asked to update their phone number')
```

Comment:
Looks like this event tracker is not present in the new EditInfoController. Could you please add it there so we can continue to track this event?